style.css

```diff
@@ -482,4 +488,476 @@
+    background-color: #396df2;
+  }
+
+  .btns:active {
+    color: #fff;
+    background-color: #396df2;
+  }
+
+  .pang {
+    display: flex;
+    list-style-type: none;
+    gap: 30%;
+  }
+
+  .lang-p {
+    font-family: "Poppins", sans-serif;
+    font-size: 15px;
+    font-weight: 500;
+    line-height: 20px;
+    letter-spacing: 0;
+    text-align: left;
+  }
+
+  hr {
+    width: 200%;
+  }
+
+  h3 {
+    height: 36px;
+    width: 295px;
+    font-family: "Poppins", sans-serif;
+    font-size: 32px;
+    font-weight: 700;
+    line-height: 44px;
+    letter-spacing: 0;
+    text-align: left;
+    color: #172b40;
+  }
+
+  .lang-hd {
+    font-family: "Poppins", sans-serif;
+    font-size: 20px;
+    font-weight: 500;
+    line-height: 24px;
+    letter-spacing: 0;
+    text-align: left;
+  }
+
+  .fas {
+    margin-left: 190%;
+  }
+
+  .formsection {
+    border-top-left-radius: 90px;
+    background-image: url(img/contact\ form\ background\ shapes@2x.png);
+    background-size: 100%;
+    background-color: #6070ff;
+    width: 1354px;
+    background-repeat: no-repeat;
+    background-position-x: 85%;
+    height: 826px;
+    margin-left: -1%;
+    margin-bottom: -1%;
+  }
+
+  .Fsection {
+    padding-top: 1%;
+  }
+
+  .contact {
+    font-family: "Poppins", sans-serif;
+    font-size: 40px;
+    font-weight: 700;
+    line-height: 52px;
+    letter-spacing: 0;
+    text-align: center;
+    color: #ebebff;
+    margin-top: 90px;
+    margin-bottom: -10px;
+  }
+
+  .tech-r {
+    padding-top: 2px;
+    font-family: "Poppins", sans-serif;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 16px;
+    letter-spacing: 0.03em;
+    list-style-type: none;
+    text-align: center;
+    color: #6070ff;
+    width: 98px;
+    height: 24px;
+    background: #ebebff;
+    border-radius: 8px;
+  }
+
+  .pcontact {
+    height: 102px;
+    width: 569px;
+    font-family: "Poppins", sans-serif;
+    font-size: 20px;
+    font-weight: 400;
+    line-height: 28px;
+    letter-spacing: 0;
+    text-align: center;
+    color: #ebebff;
+    margin-left: 29%;
+    margin-top: 5%;
+  }
+
+  .form {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+  }
+
+  .mail {
+    margin-top: 15px;
+    height: 48px;
+    width: 447px;
+    left: 16px;
+    top: 15px;
+    border: 1px solid #cfd8dc;
+  }
+
+  .btn {
+    color: #6070ff;
+    background-color: #fff;
+    border-radius: 8px;
+    padding: 12px 25px;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 17px;
+    line-height: 24px;
+    border: 1px;
+    margin-top: 20px;
+    cursor: pointer;
+    margin-bottom: 5px;
+    width: 10%;
+  }
+
+  .btn:hover {
+    background-color: #6070ff;
+    color: #fff;
+    border: 1px solid #fff;
+  }
+
+  textarea {
+    margin-top: 15px;
+    width: 447px;
+    left: 16px;
+    top: 15px;
+    border: 1px solid #cfd8dc;
+  }
+
+  .name {
+    height: 48px;
+    width: 447px;
+    border: 1px solid #cfd8dc;
+  }
```

Comment:
- [ ] Please check the design requirements provided by the Figma design and apply them to your form input placeholders. 🤝 
- [ ] Please reduce the space between the contact header text and the paragraph. ⏫ 

### Your design ⬇️ 
![image](https://github.com/Danny-imo/my-project/assets/117144028/9ddcd79e-b830-4c9a-9bdd-065651898bcf)

### Figma design ⬇️ 
![image](https://github.com/Danny-imo/my-project/assets/117144028/f51ed223-f89c-48c1-803b-97f21ef6aea1)